public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -329,7 +329,11 @@ export class LokiDatasource
   query(request: DataQueryRequest<LokiQuery>): Observable<DataQueryResponse> {
     const queries = request.targets
       .map(getNormalizedLokiQuery) // used to "fix" the deprecated `.queryType` prop
-      .map((q) => ({ ...q, maxLines: q.maxLines ?? this.maxLines }));
+      .map((q) => ({
+        ...q,
+        maxLines: q.maxLines ?? this.maxLines,
+        scopes: config.featureToggles.scopeFilters ? request.scopes?.flatMap((scope) => scope.spec.filters) : undefined,
```

Comment:
We might want to use a logQLScope feature toggle here, so we don't have to disable scopes everywhere in case this breaks